app/src/main/res/layout/activity_daily_word.xml

```diff
@@ -0,0 +1,127 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:minHeight="0dp"
+    tools:context="com.example.jasmiensofiecels.wordly.view.dailyWord.DailyWordActivity"
+    tools:layout_editor_absoluteY="81dp">
+
+
+        <!--Search word Multi-Auto Complete -->
+    <LinearLayout
+        android:id="@+id/search_layout_linear"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content">
+
+
+        <EditText
+            android:id="@+id/search_word_et"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_weight="1"
+            android:paddingStart="10dp"
+            android:paddingEnd="10dp"
+            android:hint="Search"
+            android:gravity="center_horizontal"
+            android:digits="ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz-"
+            />
+
+        <com.github.clans.fab.FloatingActionButton
+            android:id="@+id/search_fab"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_gravity="end"
+            android:layout_toRightOf="@+id/search_word_et"
```

Comment:
I know I'm being really pedantic at this point but you want to use `start` and `end` over `left` and `right` in case of languages where layouts are right to left.